<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadInputBox</class>
 <widget class="QWidget" name="ReadInputBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReadInputBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
		border-width: 1px;
		border-radius: 5px;
		border-color: rgb(255, 3, 62);
		border-style: solid;
}</string>
  </property>
  <widget class="QGroupBox" name="pnv_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>ПНВ</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>32</y>
      <width>351</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="read_area_btn">
         <property name="text">
          <string>Участки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="read_pnv_btn">
         <property name="text">
          <string>ПНВ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="area_line"/>
       </item>
       <item>
        <widget class="QLineEdit" name="pnv_line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="area_label">
         <property name="text">
          <string>Не загружено</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pnv_label">
         <property name="text">
          <string>Не загружено</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="angp_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Наведение</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="read_angp_btn">
       <property name="text">
        <string>Траектории</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="our_trace_line"/>
       </item>
       <item>
        <widget class="QLineEdit" name="target_trace_line">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="our_trace_label">
         <property name="text">
          <string>Не загружено</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="target_trace_label">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Не загружено</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
